bokehjs/test/unit/core/util/array.ts

```diff
@@ -109,4 +109,42 @@ describe("core/util/array module", () => {
     expect(array.split([1, null, 2], null)).to.be.equal([[1], [2]])
     expect(array.split([0, 1, null, 2, null, 3, 4, 5], null)).to.be.equal([[0, 1], [2], [3, 4, 5]])
   })
+
+  it("linspace should return an array of a given length between two given numbers", () => {
+    expect(array.linspace(2, 3, 5)).to.be.equal([ 2, 2.25, 2.5, 2.75, 3 ])
+    expect(array.linspace(2, 3).length).to.be.equal(100)
+    expect(array.linspace(2, 3, 0)).to.be.equal([])
+    expect(array.linspace(2, 3, NaN)).to.be.instanceof(RangeError)
```

Comment:
This would work if `linspace()` returned `RangeError`, but it throws it, so one should use this:
```ts
expect(() => array.linspace(2, 3, NaN)).to.throw(RangeError)
```

Reply:
Thank you for your input (here and on the other chunks of code)! I was looking for something like `to.throw` but could not find it..I guess my brian is a bit too wired using Jest and Playwright 😄